chi/lease.py

```diff
@@ -592,14 +594,23 @@ def add_network_reservation(
     if vswitch_name:
         desc_parts.append(f"VSwitchName={vswitch_name}")
 
+    if physical_network != DEFAULT_PHYSICAL_NETWORK:
```

Comment:
maybe we check if `physical_network` is defined. If it is defined, use whatever user put there; otherwise, don't set `resource_properties` so that it will use the blazer defaults.